### DISCUSSION TITLE
CartesianChart with vertical lines to mark errors in data

### DISCUSSION THREAD

**Discussion by mjanulaitis** [Live-Charts/LiveCharts2#609] [General]
I am currently plotting ping time span results to a CartesianChart (with a branched version of LiveCharts v1).  I would like to include vertical lines that extend from the bottom of the chart to the top when ever I have a timeout in my data showing all the errors.  This format allows the user to see time periods where the network latency is high.  Any idea how this can be accomplished with LiveCharts2?

**Comment by Martin-Yuan** [comment]
you can try use LiveChart v2 ,the part of Sections
